junos/resource_static_route_test.go

```diff
@@ -188,6 +241,19 @@ resource junos_static_route testacc_staticRoute_ipv6_instance {
   }
   community = ["no-advertise"]
 }
+resource junos_static_route testacc_staticRoute_ipv6_interface {
+  destination      = "2001:db8:85a2::/48"
+  routing_instance = junos_routing_instance.testacc_staticRoute.name
+  preference       = 102
+  metric           = 102
+  qualified_next_hop {
+	next_hop = "2001:db8:85a4::1"
+	preference = 102
+	metric = 102
+	interface = "st0.0"
+  }
+  community = ["no-advertise"]
+}
```

Comment:
Same comment as for ipv4. Just add second `qualified_next_hop` to the existing resource

Reply:
Fixed